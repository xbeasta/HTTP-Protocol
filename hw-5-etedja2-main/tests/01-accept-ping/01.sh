#!/bin/bash

PORT=$@

telnet 127.0.0.1 $PORT < /dev/null
echo -n ""