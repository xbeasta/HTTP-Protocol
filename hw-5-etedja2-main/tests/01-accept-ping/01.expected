Trying 127.0.0.1...
Connected to 127.0.0.1.
Escape character is '^]'.
Connection closed by foreign host.
